src/ShaderGen.App/Program.cs

```diff
@@ -519,6 +522,26 @@ private static string FindFxcExe()
             }
 
             return null;
+        }
+
+        private static void GetXcodePath()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                ProcessStartInfo startInfo = new ProcessStartInfo
+                {
+                    FileName = "xcode-select", // Specify exe name.
+                    Arguments = "--print-path",
+                    RedirectStandardOutput = true
+                };
+                using (Process process = Process.Start(startInfo))
+                {
+                    using (StreamReader reader = process.StandardOutput)
+                    {
+                        s_xcodePath = reader.ReadLine();

```

Comment:
The logic before was resilient to xcode not being installed. I believe this is going to throw an exception now if you don't have it installed.

Reply:
My assumption was if Mac OS should have Xcode :), will improve as suggested :)